app/models/host_status/configuration_status.rb

```diff
@@ -77,6 +77,10 @@ def to_status(options = {})
       end
     end
 
+    def relevant?
```

Comment:
Kind of unrelated but if you could add a comment on `HostStatus::Status#relevant?` explaining what that method is used for that'd be helpful. It's hard to understand, even in context.